### PR TITLE
Add CD audio capture buffers

### PR DESCRIPTION
**Pull request by KieronJ**
Adding the capture buffers gets CTR in-game, and might help other games that use CD audio.

Replaced tabs in noise files with 4 spaces.

### CODE REVIEW

File: src/device/spu/spu.cpp [hunk 1]
```diff
@@ -131,15 +137,13 @@ void SPU::step(device::cdrom::CDROM* cdrom) {
         bufferReady = true;
     }
 
-    if (control.irqEnable && irqAddress._reg < 0x200) {
-        static int cnt = 0;
+    uint32_t cd_left_address = capture_buffer_index;
```
Comment (JaCzekanski) [inline src/device/spu/spu.cpp]: Please change to camelCase - I know it's not c++ like, but that's the convention I'm enforcing in this project.

I'll add style guide to CONTRIBUTING.md someday.

File: src/device/spu/noise.cpp [hunk 2]
```diff
@@ -3,40 +3,40 @@
 
 namespace {
 std::array<int8_t, 64> noise_addition = {
-	{1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0,
-	 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0,
-	 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1,
-	 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1}};
+    {1, 0, 0, 1, 0, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 0,
```
Comment (JaCzekanski) [inline src/device/spu/noise.cpp]: You can use clang-format to take care of these things.

Put [this](https://gist.github.com/JaCzekanski/b13ae5d839dfc2fd3029d21e2cd5879b) in .git/hooks/pre-commit to do it when commiting changes.

**Review by JaCzekanski — APPROVED** [review]

File: src/device/spu/spu.cpp (hunk 1 above)
Comment (KieronJ) [reply src/device/spu/spu.cpp]: Alright, all changes are now done.

File: src/device/spu/noise.cpp (hunk 2 above)
Comment (KieronJ) [reply src/device/spu/noise.cpp]: Thanks for the tip.

**Review by JaCzekanski — APPROVED** [review]
Looks great!

**Pull request merged**